lib/BaseStripeClient.php

```diff
@@ -186,7 +186,10 @@ public function rawRequest($method, $path, $params = null, $opts = [])
             $headers['Stripe-Context'] = $opts['stripe_context'];
             unset($opts['stripe_context']);
         }
-        $opts = $this->defaultOpts->merge($opts, true);
+
+        $defaultRawRequestOpts = $this->defaultOpts->merge(['stripe_version' => \Stripe\Util\ApiVersion::PREVIEW], true);
```

Comment:
Shouldn't we only set `'stripe_version'` to the preview version if `'preview' === $apiMode`?

Reply:
Yes, I'll add another test